yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -12,375 +12,181 @@
+        </t>
+      </section>
+    </section>
+    <section anchor="terminology" title="Terminology and Conventions">
+      <t>The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14 <xref target="RFC2119"/>
+        <xref target="RFC8174"/> when, and only when, they appear in all capitals, as shown here.
+      </t>
+      <t>This document makes use of the following terminology introduced in the YANG 1.1 Data Modeling Language <xref target="RFC7950"/>:
+        <list style="symbols">
+          <t>schema node</t>
+        </list>
+      </t>
+      <t>In addition, this document uses the following terminology:
+        <list style="symbols">
+          <t>YANG module revision: An instance of a YANG module, uniquely identified with a revision date, with no implied ordering or backwards compatibility between different revisions of the same module.</t>
+          <t>Backwards-compatible (BC) change: A backwards-compatible change between two YANG module revisions, as defined in
+            <xref target="bc_update_rules"/></t>
+          <t>Non-backwards-compatible (NBC) change: A non-backwards-compatible change between two YANG module revisions, as defined in <xref target="nbc_update_rules"/></t>
+        </list>
+      </t>
+    </section>
+    <section anchor="yang_revision_handling" title="Refinements to YANG revision handling">
+      <t><xref target="RFC7950"/> and <xref target="RFC6020"/> assume, but do not explicitly state, that the revision history for a YANG module or submodule is strictly linear, i.e., it is prohibited to have two independent revisions of a YANG module or submodule that are both directly derived from the same parent revision.</t>
+      <t>This document clarifies <xref target="RFC7950"/> and <xref target="RFC6020"/> to explicitly allow non-linear development of YANG module and submodule revisions, so that they MAY have multiple revisions that directly derive from the same parent revision.  As per <xref target="RFC7950"/> and <xref target="RFC6020"/>, YANG module and submodule revisions continue to be uniquely identified by their revision date, and hence all revisions of a given module or submodule MUST have unique revision dates.</t>
+      <t>However, using revision dates alone to identify revisions of a YANG module versioned with a branched revision history is likely to be confusing because the relationship between module revisions is no longer guaranteed to be chronologically ordered. Instead, for modules that may use a branched revision history, it is RECOMMENDED to use a version identifier, such as the one described in <xref target="I-D.ietf-netmod-yang-semver"/>, better describes the semantic relationship between the revisions.</t>
```

Comment:
change "better" to "that better"  (e.g. picture the flow of the sentence without the "such as" clause).

Reply:
I also spotted and fixed this in my review.